serverless/src/util/cmr/__tests__/pageAllCmrResults.test.js

```diff
@@ -70,6 +70,6 @@ describe('pageAllCmrResults', () => {
     })
 
     expect(consoleMock).toBeCalledTimes(1)
-    expect(consoleMock.mock.calls[0]).toEqual(['Error (400): HTTP Error'])
+    expect(consoleMock.mock.calls[0]).toEqual(['AxiosError (400): HTTP Error'])
```

Comment:
I'm not sure if we should update these 
`/Users/eaolivar/git/earthdata-search/serverless/src/fetchCatalogRestOrder/__tests__/handler.test.js` and 
/Users/eaolivar/git/earthdata-search/sharedUtils/__tests__/parseError.test.js
I kind of lean towards no but, I'd be curious to hear other opinions. Since these are all mocks and the change is really not meaningfully helpful to us

Reply:
If we don't update these the test will fail?

Reply:
No they won't. I think from the slack in thread we intend to keep those as they are now